### DISCUSSION TITLE
[Unknown Mod][Spells] Tweaks to Anti-Magic spell, mirror image troop [Removed]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2638] [Ideas]
I would like to be able to cast the Anti-Magic spell on a mirror troop to protect it from the enemy hero's magic attacks.

This will be an option:
"combat: allow to cast the Anti-Magic spell on a mirror image troop"

**Comment by Branikolog** [comment]
Well... It's about changing the spell's rules... It seems logically ok, when we cannot cast anti-magic over a magic image of creature...
Also, I count this spell really strong, so making it unable to be killed my magic attack seems to me quite a imbalance feature...
But who knows? Maybe people would like to see such feature..

**Reply by LeHerosInconnu** [reply]
This will be part of my mod, so yes, of course there will be some changes from the original game, that's the goal. :)

**Comment by LeHerosInconnu** [comment]
Removed from the mod.
I close the discussion.
